app.js

```diff
@@ -3,6 +3,23 @@ var express = require('express');
 var path = require('path');
 var cookieParser = require('cookie-parser');
 var logger = require('morgan');
+var session = require('express-session');
+let randomString = require('./utils/utils');
+let { v4: uuidv4 } = require('uuid');
```

Comment:
What is this used for ?

Reply:
I used it earlier but since not using it now so removed it.